docs/getting_started/index.rst

```diff
@@ -0,0 +1,167 @@
+===============
+Getting Started
+===============
+The following directions will walk through the process of deploying Scribe and using the rsync functionality.
+In the future, this document will also contain steps for the rclone functionality.
+
+Deploying
+=========
+Once the repository has been cloned run the following within the scribe directory to deploy the required
+components.
+
+Volume SnapShot capabilities are required for Scribe. If you are using Kind to develop functionality for
+Scribe or another Kubernetes provider ensure that you are using a CSI capable storageclass.
+
+If you are wanting try scribe run the make command to deploy the objects into
+the cluster. Before starting ensure that your KUBECONFIG is set to the cluster you want to install Scribe.
+
+.. code-block:: bash
+
+   make deploy
+
+Verify Scribe is running.
```

Comment:
nit, but I'd like to see the correct output of the command so I know I have podA and podB running or supposed to be running

```
  $ kubectl get pods -n scribe-system
  NAME                                         READY   STATUS    RESTARTS   AGE
  scribe-controller-manager-646bd9d4d9-trzqb   2/2     Running   0          115s
```